models/containerized_model/README.md

```diff
@@ -185,3 +185,56 @@ This should create a docker container and host the webservice.
 
 ### Step 6 - Test
 Make a request to `http://127.0.0.1:8551/predict` with the respective parameters.
+
+
+## [Proxy Template](#proxy-template)
+### Step 1 - Template generation
+
+Generate the code template for containerization of your model:
+```
+./generate.sh -d generated-container-proxy -i certifai-proxy-container:latest -m proxy -n containerproxy -r container-proxy
+```
+
+This command should create a directory called `generated-container-proxy`
+in your current directory with the generated code.
+
+For more `generate` options:
+```
+./generate.sh --help
+```
+
+### Step 2 - Copy artifacts
+Copy the `packages` folder from inside the toolkit into the generated
+directory `generated-container-proxy`:
+
+```
+cp -r <certifai-toolkit-path>/packages generated-container-proxy/packages
+```
+
+### Step 3 - Configure hosted model url 
+Add `HOSTED_MODEL_URL`  env variable to `generated-container-proxy/environment.yml` file. This will be used in the `RUN` step.
+
+Optionally, add any additional auth/secret header token to above file. Don't forget to reference the same additional env variable in `src/prediction_service.py`
```

Comment:
Should the env file template include HOSTED_MODEL_AUTH_HEADER_TOKEN? 

When there's nothing in the env file, I'm guessing the proxy is sending auth header of "Bearer None"?

Reply:
made it optional and added to the environment.yml file with one line helper docstring. fixed